Doc/reference/executionmodel.rst

```diff
@@ -133,14 +133,15 @@ determined by scanning the entire text of the block for name binding operations.
 See :ref:`the FAQ entry on UnboundLocalError <faq-unboundlocalerror>`
 for examples.
 
-If the :keyword:`global` statement occurs within a block, all uses of the names
+The :keyword:`global` statement must precede all uses of the listed names.
```

Comment:
I don't think this change is an improvement; the old wording flowed better.

Reply:
Would you like me to put this sentence back to the end of the paragraph like it was in the original version?

Reply:
Yes. I thought the previous text was almost ready to go but further changes are unnecessary.